### DISCUSSION TITLE
Find some tips

### DISCUSSION THREAD

**Discussion by VeriJoly** [Lakhankumawat/LearnCPP#1577] [General]
Where should I start learn C++
P/s: i'm new at this

**Comment by Lakhankumawat** [comment]
It depends what you want to use c++ for

**Comment by Lakhankumawat** [comment]
If you are using c++ for data structures , algorithms and competitive coding then i will recommend you go with c++ or java both of them works like a charm .

You can also use c++ for compiler design .

There can be many more uses but i've only used c++ in the above listed 2 .
